setup.py

```diff
@@ -121,6 +122,121 @@
     'Topic :: Scientific/Engineering :: Visualization',
     ]
 
+
+class NoseTestCommand(TestCommand):
+    """Invoke unit tests using nose after an in-place build."""
```

Comment:
What do you mean by "an in-place build" here?  Does this mean one can't run the tests after a normal `python setup.py install`?